### ISSUE TITLE
Does not work with latest Firefox version (37.0)

### ISSUE THREAD

**Issue by nathanielw** [prikhi/pencil#685]
Someone on reddit [just pointed out](http://www.reddit.com/r/androiddev/comments/31rd0l/easy_android_prototyping_with_pencilgui/cq5c62i?context=10000) that Pencil doesn't run with version 37.0 of Firefox, due to the application.ini having 36.\* set as the max version. I can confirm that after changing the MaxVersion it runs fine with Firefox 37. (I'm not sure what the best practice is regarding what the MaxVersion should be set to for Firefox extensions/apps, so haven't made a PR)

**Comment by Zyzle** [comment]
I've noticed this in testing too. If the FF project is using semantic versioning then setting it to 37.\* should be correct as nothing in the 37 version line should have breaking changes.

**Comment by prikhi** [comment]
I checked out the [release docs](https://developer.mozilla.org/en-US/Firefox/Releases/37) and it doesn't look like there's any breaking changes(breaking changes aren't separated out, usually have to search the page for `breaking` or `incompatible`).

So I've bumped it & will put out a new release & upload new builds for `2.0.9`.

As for PRs, I'd like to keep `MaxVersion` on a major version that has been tested(instead of just doing `MaxVersion=*`). The bare minimum tests I do are:
- Can I load a file?
- Can I save, quit then load without losing data?
- Can I export to PNG?

If you attest to testing these things, I'll take a PR that bumps `MaxVersion`.
